react/features/display-name/components/web/StageParticipantNameLabel.tsx

```diff
@@ -38,8 +37,8 @@ const useStyles = makeStyles<IOptions, 'screenSharing'>()((theme, options: IOpti
         // the clientHeight will be 960px if there are some titlebars, toolbars, addressbars, etc visible.For any other
         // screen size we will decrease/increase the font size based on the screen size.
 
-        typography.fontSize = scaleFontProperty(clientHeight, getStageParticipantFontSizeRange(theme));
-        typography.lineHeight = getStageParticipantNameLabelLineHeight(theme, clientHeight);
+        typography.fontSize = `${scaleFontProperty(clientHeight, getStageParticipantFontSizeRange(theme))}rem`;
+        typography.lineHeight = `${getStageParticipantNameLabelLineHeight(theme, clientHeight)}rem`;
```

Comment:
If I read this correctly here we should endup with `remrem` because `getStageParticipantNameLabelLineHeight` returns string with `rem` in it.